### PR TITLE
chore: add static modal stories for visual tests

### PR DESCRIPTION
**Pull request by dtsanevmw**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Adds static modal stories for visual testing.

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: .storybook/stories/modal/modal-static.stories.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { ClrModal, ClrModalModule } from '@clr/angular';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+    <style>
+        .backdrop-example-container {
```
Comment (kevinbuhmann) [inline .storybook/stories/modal/modal-static.stories.ts:16]: nit: double indentation in this template

File: .storybook/stories/modal/modal-static.stories.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            position: relative;
+            padding: $clr_baselineRem_1;
```
Comment (kevinbuhmann) [inline .storybook/stories/modal/modal-static.stories.ts:18]: Does this work in an inline `style` tag? Neat.

File: .storybook/stories/modal/modal-static.stories.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            position: absolute;
+            top: 0;
+            right: 0;
+            bottom: 0;
+            left: 0;
+        }
+    </style>
+    <div class="backdrop-example-container">
+        <div class="modal static">
+        <div class="modal-dialog modal-{{size}}" role="dialog" aria-hidden="true">
```
Comment (kevinbuhmann) [inline .storybook/stories/modal/modal-static.stories.ts:35]: nit: missing indentation

File: .storybook/stories/modal/modal-static.stories.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        </div>
+        <div class="modal-backdrop static" aria-hidden="true"></div>
+    </div>
+  `,
+  props: { ...args },
+});
+
+const defaultParameters: Parameters = {
+  title: 'Modal/Static',
+  component: ClrModal,
```
Comment (kevinbuhmann) [inline .storybook/stories/modal/modal-static.stories.ts:58]: This should be removed since it is not using the component.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]
minor things

File: .storybook/stories/modal/modal-static.stories.ts (hunk 2 above)
Comment (dtsanevmw) [reply .storybook/stories/modal/modal-static.stories.ts:18]: Yeah, missed that. Thanks!

File: .storybook/stories/modal/modal-static.stories.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        left: 0;
+      }
+    </style>
+    <div class="backdrop-example-container">
+        <div class="modal static">
+            <div class="modal-dialog modal-{{size}}" role="dialog" aria-hidden="true">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button aria-label="Close" class="close" type="button">
+                            <cds-icon aria-hidden="true" shape="window-close"></cds-icon>
```
Comment (kevinbuhmann) [inline .storybook/stories/modal/modal-static.stories.ts:39]: This and the other button still has double indentation. I will approve this though and you can fix it if you want.

**Review by kevinbuhmann — APPROVED** [review]

File: .storybook/stories/modal/modal-static.stories.ts (hunk 2 above)
Comment (kevinbuhmann) [reply .storybook/stories/modal/modal-static.stories.ts]: I was pretty sure it wouldn't work, lol. Thanks for fixing it.

File: .storybook/stories/modal/modal-static.stories.ts (hunk 5 above)
Comment (dtsanevmw) [reply .storybook/stories/modal/modal-static.stories.ts:39]: Had a wrong configuration for spaces. Should be good now :)

**Pull request merged**